Rename 'developer/development credentials' to 'user credentials' (#9500)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/exports.md

## Exports in development

> You can run an export in your development environment using your user credentials if you want to test the output of the export before production. 

Reviewer: I think the wording here is confusing even before the change
```suggestion
You can run an export in your development environment if you want to test its output before production. 
```
